paddle/operators/conv_op.h

```diff
@@ -350,5 +351,69 @@ class GemmConvGradKernel : public framework::OpKernel<T> {
     }
   }
 };
+
+template <typename DeviceContext, typename T>
+class DepthwiseConvKernel : public framework::OpKernel<T> {
+ public:
+  void Compute(const framework::ExecutionContext& context) const override {
+    const Tensor* input = context.Input<Tensor>("Input");
+    Tensor filter = *context.Input<Tensor>("Filter");
+    Tensor* output = context.Output<Tensor>("Output");
+    output->mutable_data<T>(context.GetPlace());
+
+    std::vector<int> strides = context.Attr<std::vector<int>>("strides");
+    std::vector<int> paddings = context.Attr<std::vector<int>>("paddings");
+    std::vector<int> dilations = context.Attr<std::vector<int>>("dilations");
```

Comment:
Doesn't `DepthwiseConv` support `groups`？

Reply:
The groups value equals to the input channels num.

Reply:
I add the choice to the python code, it will execute the depthwise conv op when groups size equals to the input channels. But i don't add any check on the c++ side,  because i  directly use the  conv op instead recreate one.